pages/workspace/workspace.js

```diff
@@ -22,6 +26,11 @@ router.get('/:workspace_id/:action', asyncHandler(async (req, res, next) => {
         logger.info(`Rebooting workspace ${workspace_id}.`);
         await workspaceHelper.updateState(workspace_id, 'stopped', 'Rebooting container');
         res.redirect(`/pl/workspace/${workspace_id}`);
+    } else if (action === 'reset') {
+        logger.info(`Resetting workspace ${workspace_id}.`);
```

Comment:
I think we should change all `logger.info()` calls in this file to `debug()`, to avoid spamming the logs in production.